src/containerapp/azext_containerapp/_validators.py

```diff
@@ -49,6 +50,13 @@ def validate_create(registry_identity, registry_pass, registry_user, registry_se
         raise InvalidArgumentValueError("--registry-identity: expected an ACR registry (*.azurecr.io) for --registry-server")
 
 
+def validate_runtime(runtime, enable_java_metrics):
+    if runtime and runtime.lower() not in SUPPORTED_RUNTIME_LIST:
+        raise ValidationError(f"Runtime {runtime} is not supported. Supported runtimes are {', '.join(SUPPORTED_RUNTIME_LIST)}.")
+    if runtime and runtime.lower() == RUNTIME_GENERIC and enable_java_metrics is not None:
+        raise ValidationError("Java metrics are not supported for generic runtime.")

```

Comment:
Please specify the arguments, for example:

Not support enable Java metrics with --enable-java-metrics for generic runtime with --runtime generic